docs/sphinx/using/simulators.rst

```diff
@@ -80,38 +80,90 @@ simulator that will be used.
 * **`OMP_PLACES=cores`**: Set this environment variable to improve CPU parallelization.
 * **`OMP_NUM_THREADS=X`**: To enable CPU parallelization, set X to `NUMBER_OF_CORES_PER_NODE/NUMBER_OF_GPUS_PER_NODE`.
 
-A note on **CUDA_VISIBLE_DEVICES**: This environment variable should **always** be set before using the :code:`tensornet` 
-backend if you have multiple GPUs available. With OpenMPI, you can run a multi-GPU quantum circuit simulation like this:
+.. note:: 
 
-.. code:: bash 
+    The **CUDA_VISIBLE_DEVICES** environment variable should **always** be set before using the :code:`tensornet` 
+    backend if you have multiple GPUs available. With OpenMPI, you can run a multi-GPU quantum circuit simulation like this:
+
+    .. code:: bash 
     
-    mpiexec -n 8 sh -c 'CUDA_VISIBLE_DEVICES=${OMPI_COMM_WORLD_LOCAL_RANK} binary.x > tensornet.${OMPI_COMM_WORLD_RANK}.log'
+        mpiexec -n 8 sh -c 'CUDA_VISIBLE_DEVICES=${OMPI_COMM_WORLD_LOCAL_RANK} binary.x > tensornet.${OMPI_COMM_WORLD_RANK}.log'
+
+    This command will assign a unique GPU to each MPI process within the node with 8 GPUs and produce a separate output for each MPI process.
+
+`cuTensorNet` matrix product state 
++++++++++++++++++++++++++++++++++++
 
-This command will assign a unique GPU to each MPI process within the node with 8 GPUs and produce a separate output for each MPI process.
+The :code:`tensornet-mps` backend is based on the matrix product state (MPS) representation of the state vector/wave function, exploiting the sparsity in the tensor network via tensor decomposition techniques such as QR and SVD. As such, this backend is an approximate simulator, whereby the number of singular values may be truncated to keep the MPS size tractable. 
 
-To specify the use of the :code:`tensornet` target, pass the following command line 
+This backend exposes a set of environment variables to configure specific aspects of the simulation:
+
+* **`CUDAQ_MPS_MAX_BOND=X`**: The maximum number of singular values to keep (fixed extent truncation). Default: 64.
+* **`CUDAQ_MPS_ABS_CUTOFF=X`**: The cutoff for the largest singular value during truncation. Eigenvalues that are smaller will be trimmed out. Default: 1e-5.
+* **`CUDAQ_MPS_RELATIVE_CUTOFF=X`**: The cutoff for the maximal singular value relative to the largest eigenvalue. Eigenvalues that are smaller than this fraction of the largest singular value will be trimmed out. Default: 1e-5
+
+The :code:`tensornet-mps` only supports single-GPU simulation. Its approximate nature allows the :code:`tensornet-mps` backend to handle a large number of qubits for certain classes of quantum circuits on a relatively small memory footprint.
+
+.. warning:: 
+
+    The :code:`tensornet-mps` cannot handle quantum gates acting on more than two qubit operands. It will throw an error when this constraint is not satisfied.
```

Comment:
I thought I also saw a similar commend and some disabled tests for the tensornet backend. If so, there should be a similar note in the section above for the `tensornet` target.

Reply:
We disabled those tests involving 3+ qubit gates only for the `tensornet-mps` backend.
We also disabled tests involving checking the random seed and tests that have long runtime for both backends. 
I've added a [ticket](https://github.com/NVIDIA/cuda-quantum/issues/885) to address this.